### PR TITLE
Basic framework for the B2B trial barrier

### PR DESCRIPTION
**Pull request by paddyo**
## Description
Adds the basic functionality for the B2B Trail barrier.

## Feature Flag
`signupFormB2B`

### CODE REVIEW

File: views/partials/barriers/corporate-trial.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+		<div class="barrier__container" data-o-grid-colspan="12 L10 center">
+			<div class="barrier-grid" data-trackable="corporate-trial"
+				data-barrier="corporate-trial"
+				data-barrier-is-product-selector="false">
+
+				<div class="barrier-grid__header o-grid-row o-grid-row--compact">
+					<h2>Trial Barrier</h2>
+					<p class="barrier-grid__sign-in">
+						<span class="barrier-grid__pretext">Already a subscriber?</span>
+						<a data-trackable="sign-in" href="/login">Sign	In</a>
```
Comment (phamann) [inline views/partials/barriers/corporate-trial.html]: Double space

**Comment by phamann** [comment]
tests?

**Comment by phamann** [comment]
👍 Other than comments

**Comment by paddyo** [comment]
The tests do not run on this project.

**Pull request merged**